spec/features/01_next_up_spec.rb

```diff
@@ -0,0 +1,179 @@
+require "rails_helper"
+
+describe "The Next Up section" do
+  it "has a div element with the class \"next_up\"", points: 1 do
```

Comment:
What do you think about moving this spec earlier in the sequence?

- The homepage has a div with class "next_up"
- The homepage has an h2 with text "Next Up" within the div of class "next_up"
- etc for the other two sections

That may help them structure the page correctly before getting into the weeds too far?